src/ServiceControl/Hosting/Commands/ImportFailedErrorsCommand.cs

```diff
@@ -28,10 +28,10 @@ public override async Task Execute(HostArguments args, Settings settings)
             // TODO: Ideally we would never want to actually bootstrap the web api. Figure out how
             var hostBuilder = WebApplication.CreateBuilder();
             hostBuilder.AddServiceControl(settings, endpointConfiguration, loggingSettings);
-            using var app = hostBuilder.Build();
 
+            using var app = hostBuilder.Build();
```

Comment:
Should we remove the `using` here to keep it aligned with the various other places where we create, start, and stop apps?

Reply:
@mauroservienti when you are not using `RunAsync` you have to dispose manually